src/main/kotlin/com/cognifide/gradle/aem/environment/docker/runtime/Toolbox.kt

```diff
@@ -10,10 +10,21 @@ class Toolbox(aem: AemExtension) : Base(aem) {
         get() = NAME
 
     override val hostIp: String
-        get() = aem.props.string("environment.docker.toolbox.hostIp") ?: "192.168.99.100"
+        get() = detectHostIp() ?: aem.props.string("environment.docker.toolbox.hostIp") ?: "192.168.99.100"
+
+    @Suppress("TooGenericExceptionCaught")
+    fun detectHostIp() = try {
+        ProcBuilder("docker-machine").withArg("ip").run()
```

Comment:
new feature; this IP also might change, it could be easily grabbed ... as you can see ;)